src/components/AnonymousUsername/index.js

```diff
@@ -1,34 +1,35 @@
 import * as React from 'react';
-import {StyleSheet, Text} from 'react-native';
+import {StyleSheet, Text, TextStyle} from 'react-native';
 
 import {COLORS} from '../../utils/theme';
 import {POST_VERSION} from '../../utils/constants';
 import {fonts} from '../../utils/fonts';
 import {getOfficialAnonUsername} from '../../utils/string/StringUtils';
 
 const styles = StyleSheet.create({
-  feedUsername: {
+  feedUsername: (isFeeds) => ({
     fontFamily: fonts.inter[600],
     fontWeight: 'bold',
     fontSize: 14,
     lineHeight: 16.94,
     color: COLORS.black,
-    flex: 1
-  }
+    flex: isFeeds ? 0 : 1
+  })
 });
 
 /**
  * @typedef {Object} AnonymousUsernameComponentProps
  * @property {string} version
  * @property {AnonUserInfoTypes} anonUserInfo
+ * @property {TextStyle} style
  */
 
 /**
  *
  * @param {AnonymousUsernameComponentProps} props
  */
 const AnonymousUsername = (props) => {
-  const {version, anonUserInfo} = props;
+  const {version, anonUserInfo, isFeed} = props;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `AnonymousUsername` component now accepts an additional prop `isFeed`. This is a good addition as it allows for more flexible usage of the component. However, please make sure to update the PropTypes or TypeScript interface (if any) to include this new prop.

```diff
-  const {version, anonUserInfo} = props;
+  const {version, anonUserInfo, isFeed} = props;
```

42:
The `feedUsername` style is now being called as a function with `isFeed` passed as an argument. This is consistent with the changes made to the `feedUsername` style definition.

```diff
-  <Text testID="newVersion" style={styles.feedUsername}>
+  <Text testID="newVersion" style={styles.feedUsername(isFeed)}>
```

48:
The `style` prop is now being applied to the `Text` component when rendering "Anonymous". This allows for more customization of the component from its parent. However, please ensure that the `style` prop is optional and has a default value to prevent potential crashes if it's not provided.

```diff
-  return <Text style={styles.feedUsername}>Anonymous</Text>;
+  return <Text style={[styles.feedUsername, props.style]}>Anonymous</Text>;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->